src/develop/blend_gui.c

```diff
@@ -1355,23 +1376,27 @@ gboolean blend_color_picker_apply(dt_iop_module_t *module, GtkWidget *picker, dt
     dt_develop_blend_params_t *bp = module->blend_params;
 
     const int tab = data->tab;
-    float *raw_mean, *raw_min, *raw_max;
-    float picker_mean[8], picker_min[8], picker_max[8];
+    float raw_min[4], raw_max[4];
+    float picker_min[8], picker_max[8];
```

Comment:
Adding DT_ALIGNED_PIXEL to these declarations will allow the compiler to use faster aligned vector load/store instructions, and will prevent crashes if these are passed to a function which internally declares its arguments to be aligned (without, it depends on how the variables wind up in the stack frame whether or not they are aligned).